DevCycle/ObjC/ObjCDVCClient.swift

```diff
@@ -136,8 +136,8 @@ public class ObjCDVCClient: NSObject {
         return variable(key: key, defaultValue: defaultValue)
     }
     
-    @objc public func boolVariable(key: String, defaultValue: Bool) -> ObjCDVCVariable {
-        return variable(key: key, defaultValue: defaultValue)
+    @objc public func boolVariable(key: String, defaultValue: NSNumber) -> ObjCDVCVariable {
+        return variable(key: key, defaultValue: defaultValue.boolValue)
```

Comment:
why is this change needed? can we do the opposite where we cast the `Bool` to a `NSNumber` in here?